ui/v2.5/src/components/Studios/StudioDetails/StudioDetailsPanel.tsx

```diff
@@ -45,9 +45,11 @@ export const StudioDetailsPanel: React.FC<IStudioDetailsPanel> = ({
         <TextField id="details" value={studio.details} />
 
         <URLField
-          id="studio"
+          id="parent_studios"
           value={studio.parent_studio?.name}
           url={`/studios/${studio.parent_studio?.id}`}
+          trusted
```

Comment:
I believe these could/should be applied in more places then. Movies for one also uses `URLField` for studio and thus also opens a new tab, and applies `noopener noreferrer`